pkg/cri/resource-manager/policy/builtin/memtier/node.go

```diff
@@ -755,6 +836,45 @@ func (n *socketnode) DiscoverMemset() {
 	}
 }
 
+// AssignNUMANodes assigns the given NUMA nodes to this one.
+func (n *socketnode) AssignNUMANodes(ids []system.ID) {
```

Comment:
looking at 3 instances of almost exact functions gives a feeling that it can be generalized into separate function?

Reply:
Updated to use a common function.